src/Core/Framework/DataAbstractionLayer/Dbal/EntityForeignKeyResolver.php

```diff
@@ -256,7 +257,10 @@ private function fetchAssociation(
             /** @var Field $pk */
             $pk = $primaryKeys->first();
             $property = $pk->getPropertyName();
-            $affected = array_column($affected, $property);
+
+            if (!Feature::has('FEATURE_NEXT_14872') || $property === 'id') {
```

Comment:
Is this flag still in the feature.yaml?

Reply:
This flag is new with my PR. Has this changed?

Reply:
nooo